app/workers/check_rules_worker.rb

```diff
@@ -3,8 +3,8 @@ class CheckRulesWorker
   sidekiq_options retry: 2
   sidekiq_options unique: :until_executing unless Rails.env.test?
 
-  def perform(workflow_id, subject_id)
+  def perform(workflow_id, subject_id, user_id)
```

Comment:
Need to make sure that the new argument is optional, so that old jobs can still run. After deploy we can have a follow-up PR that makes it non-optional.